matter/si91x/siwx917/BRD4338A/autogen/sl_event_handler.c

```diff
@@ -18,6 +19,9 @@ void sl_platform_init(void) {
   sli_si91x_platform_init();
   RSI_Board_Init();
   DEBUGINIT();
+#if SL_ICD_ENABLED && !(defined(DISPLAY_ENABLED))

```

Comment:
Did you test that SL_ICD_ENABLED exists at this level? I am not 100% sure it does

Reply:
I have tested and it is working on this level.